helper_thread.c

```diff
@@ -281,7 +281,7 @@ static void *helper_thread_main(void *data)
 		},
 		{
 			.name = "steadystate",
-			.interval_ms = steadystate_enabled ? STEADYSTATE_MSEC :
+			.interval_ms = steadystate_enabled ? ss_check_interval :
```

Comment:
In the documentation note that this forces all jobs in a single fio invocation to have the same `ss_check_interval`.

Probably we should have a check for this in `fixup_options()` to make sure that all jobs with steady state detection enabled are using the same interval.

Reply:
I can add that if you want, but that should apply to the other steadystate options too, right? (and currently there is no fixup for those)

Reply:
In `td_steadystate_init` is a check for the other options for consistency within a reporting group. Conceptually it's reasonable to run multiple jobs concurrently and allow them to have different stopping criteria.

`ss_check_interval` has to be a global option because it affects the helper thread that triggers the steady state check for all running jobs.